packages/gatsby-graphiql-explorer/src/app/app.js

```diff
@@ -142,11 +145,22 @@ const storedExplorerPaneState =
     ? window.localStorage.getItem(`graphiql:graphiqlExplorerOpen`) !== `false`
     : true
 
+const storedCodeExporterPaneState =
+  typeof parameters.codeExporterIsOpen !== `undefined`
+    ? parameters.codeExporterIsOpen === `false`
+      ? false
+      : true
+    : window.localStorage
+    ? window.localStorage.getItem(`graphiql:graphiqlCodeExporterIsOpen`) !==
+      `false`
+    : true
```

Comment:
should we default to `false` here? having 2 panes open (explorer and exporter) on first visit might hide too much of core graphiql in viewport?